app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:weightSum="1"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/album_art_size"
+        android:orientation="vertical"
+        android:background="@drawable/album_border"
+        android:padding="@dimen/album_info_border">
+
+        <TextView
+            android:id="@+id/textview_album_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:textColor="@android:color/black"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            android:padding="@dimen/album_art_border"
+            tools:text="Album Name"/>
+
+        <TextView
+            android:id="@+id/textview_artist_name_review"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_gravity="left"
+            android:layout_weight="0.5"
+            android:background="@color/colorPrimary"
+            android:gravity="center"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/list_font_size"
+            android:textStyle="bold"
+            tools:text="Artist Name"/>
+    </LinearLayout>
+
+    <ImageView
+        android:id="@+id/imageview_album_art_review"
+        android:layout_height="@dimen/album_art_size"
+        android:layout_width="@dimen/album_art_size"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_centerVertical="true"
+        android:src="@mipmap/ic_launcher"
+        android:layout_gravity="center" />
+
+    <FrameLayout
```

Comment:
i need one for the colour and one for the drawable border, i tried to put the colour in the drawable but it didnt work